jbosstools/multiple/jbosstools-multiple.target

```diff
@@ -693,6 +690,12 @@
       <unit id="net.mihai-nita.ansicon.feature.group" version="1.3.5.201612301822"/>
     </location>
 
+    <!-- Wild Web Developer -->
+    <location includeAllPlatforms="false" includeMode="slicer" includeSource="true" type="InstallableUnit">
+      <repository location="https://download.jboss.org/jbosstools/updates/requirements/wildwebdeveloper/0.8.0/"/>
```

Comment:
How come the repo is 0.8.0 when extracted feature is 9.7.2

Reply:
Did the same to me, but this is the repo used for the 2019-12 simrel : https://projects.eclipse.org/releases/2019-12
So it's just a matter of 'weird' versioning for this feature.